code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -204,38 +177,57 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorInfo'
+              example:
+                status: 409
+                code: CONFLICT
+                message: "App with a running application instance cannot be deleted"
         '500':
           $ref: '#/components/responses/500'
         '503':
           $ref: '#/components/responses/503'
  
-  /app:
-    post:
-      tags:  
+  /app/{appId}/instance:
+    post: 
+      tags:
         - Application
-      summary: Submit application metadata to the Operator Platform.                     
-      description: Contains the information about the application to be instantiated
-      operationId: submitApp
+      summary: Instantiation of an Application
+      description: Ask the OP to instantiate an application to one or several Edge Cloud Zones with an Application as an input and an Application Instance as the output. 
+      operationId: appInstantiation
+      parameters:
+        - name: appId
+          description: A globally unique identifier associated with the application. OP generates this identifier when the application is submitted over NBI.
+          in: path
+          required: true
+          schema: 
+              $ref: '#/components/schemas/AppId'
       requestBody:
-        description: The Application Provider request contains mandatory criteria (e.g. required CPU, memory, storage, bandwidth) and optional parameters. 
+        description: Details regarding where the application should be instantiated
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AppManifest'
-        required: true
+              $ref: '#/components/schemas/AppInstantiation'
```

Comment:
If we want to be able to instantiate on _several Edge Cloud Zones_ this should be an array.

Reply:
Agree with you, we have a open discussion in [https://github.com/camaraproject/EdgeCloud/discussions/215](url) and depending of the decision, we will need to adapt the input to enable the instantiation on Several Edge Cloud Zones.